src/utils/realms.ts

```diff
@@ -0,0 +1,26 @@
+import realmConfig from "@/types/RealmConfig";
+import PatientRealmModel from "@/models/realms/PatientRealm";
+import DotbaseRealmModel from "@/models/realms/DotbaseRealm";
+
+import Tokens from "@/types/Tokens";
+import JwtUtil from "./Jwt";
```

Comment:
Should be an @/ import.
```suggestion
import JwtUtil from "@/utils/Jwt";
```